src/test/java/edu/cornell/kfs/module/purap/document/service/impl/CuPurapServiceImplTest.java

```diff
@@ -241,7 +241,13 @@ private ParameterService buildMockParameterService() {
         Parameter costSourceApoOverrideParameter = buildParameterForCostSourceApoOverride();
         
         when(parameterService.getParameterValueAsString(
-                RequisitionDocument.class, PurapParameterConstants.AUTOMATIC_PURCHASE_ORDER_DEFAULT_LIMIT_AMOUNT))
+                RequisitionDocument.class, PurapParameterConstants.APO_LIMIT))

```

Comment:
Once my other comments related to `APO_LIMIT` have been addressed, please double-check this method's mock setup for `APO_LIMIT`.